addons/l10n_ro/data/account_tax_report_data.xml

```diff
@@ -1,405 +1,901 @@
-        <field name="code">ROTAX_TVA_colectata_0_Baza</field>
+    <record id="account_tax_report_ro_tva_intracom_eu" model="account.tax.report.line">
+        <field name="name">TVA COMERŢ INTRACOMUNITAR ŞI ÎN AFARA UE</field>
+        <field name="code">tax_ro_tva_intracom_eu</field>
         <field name="sequence" eval="1"/>
-        <field name="parent_id" ref="account_tax_report_baza_tva_clt"/>
         <field name="country_id" ref="base.ro"/>
     </record>
-
-    <record id="account_tax_report_baza_tva_19" model="account.tax.report.line">
-        <field name="name">Baza TVA 19%</field>
-        <field name="tag_name">Baza TVA 19%</field>
-        <field name="code">ROTAX_TVA_colectata_19_Baza</field>
+    
+    <record id="account_tax_report_ro_baza_rd1" model="account.tax.report.line">
+        <field name="name">1 - BAZA - Livrări intracomunitare de bunuri, scutite conform art. 294 alin.(2) lit.a) şid) din Codul fiscal</field>
+        <field name="tag_name">1 - BAZA - Livrări intracomunitare de bunuri, scutite conform art. 294 alin.(2) lit.a) şi d) din Codul fiscal</field>
+        <field name="code">tax_ro_baza_rd1</field>
         <field name="sequence" eval="2"/>
-        <field name="parent_id" ref="account_tax_report_baza_tva_clt"/>
+        <field name="parent_id" ref="account_tax_report_ro_baza_intracom_eu"/>
         <field name="country_id" ref="base.ro"/>
     </record>
-
-    <record id="account_tax_report_baza_tva_24" model="account.tax.report.line">
-        <field name="name">Baza TVA 24%</field>
-        <field name="tag_name">Baza TVA 24%</field>
-        <field name="code">ROTAX_TVA_colectata_24_Baza</field>
+    
+    <record id="account_tax_report_ro_baza_rd2" model="account.tax.report.line">
+        <field name="name">2 - BAZA - Regularizări livrări intracomunitare scutite conform art. 294 alin.(2) lit.a) şi d) din Codul fiscal</field>
+        <field name="tag_name">2 - BAZA - Regularizări livrări intracomunitare scutite conform art. 294 alin.(2) lit.a) şi d) din Codul fiscal</field>
+        <field name="code">tax_ro_baza_rd2</field>
         <field name="sequence" eval="3"/>
-        <field name="parent_id" ref="account_tax_report_baza_tva_clt"/>
+        <field name="parent_id" ref="account_tax_report_ro_baza_intracom_eu"/>
         <field name="country_id" ref="base.ro"/>
     </record>
-
-    <record id="account_tax_report_baza_tva_5" model="account.tax.report.line">
-        <field name="name">Baza TVA 5%</field>
-        <field name="tag_name">Baza TVA 5%</field>
-        <field name="code">ROTAX_TVA_colectata_5_Baza</field>
+    
+    <record id="account_tax_report_ro_baza_rd3" model="account.tax.report.line">
+        <field name="name">3 - BAZA - Livrări de bunuri sau prestări de servicii pentru care locul livrării/locul prestării este în afara României (în UE sau în afara UE), precum şi livrări intracomunitare de bunuri, scutite conform art. 294 alin.(2) lit.b) şi c) din Codul fiscal, din care:</field>
```

Comment:
can text for few lines(like this one) be truncated(something similar to what we have done <a href="https://github.com/odoo/odoo/blob/13.0/addons/l10n_lu/data/account_tax_report_line.xml#L295">here</a>) such that their Balance can be seen without scrolling right?
![captured](https://user-images.githubusercontent.com/7601649/69130540-b0942780-0ad6-11ea-9e8a-bf1aee993df8.gif)

Reply:
I updated and test the report with 10.000.000 amount, no need to scroll anymore.